### PR TITLE
test: refactor common tests to AVA

### PR DESCRIPTION
**Pull request by nfischer**
This is an automated PR, so double check that I got all the test cases. I'll follow up once I've verified that this translates everything correctly.

### CODE REVIEW

File: ava-test/common.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import test from 'ava';
+
+import shell from '..';
+import common from '../src/common';
+import utils from './utils/utils';
+
+test.beforeEach(() => {
+  shell.config.silent = true;
```
Comment (freitagbr) [inline ava-test/common.js]: This doesn't need to be in `beforeEach`.

Comment (nfischer) [inline ava-test/common.js]: done

**Comment by nfischer** [comment]
@freitagbr PTAL

**Comment by freitagbr** [comment]
LGTM

**Comment by nfischer** [comment]
LGTM

**Pull request merged**